### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by dmlittle**
#### What & Why:
Create the base configuration and setup to start working on the `sfmovies` repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "lint": "eslint .",
+    "release:patch": "changelog -p && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
+    "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
+    "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
+    "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
+    "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000",
+    "start": "nodemon --ignore test/"
```
Comment (elnaz) [inline package.json]: nit: This one's the generator's fault but could you order the npm scripts alphabetically.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "eslint": "^1.9.0",
+    "eslint-config-lob": "^2.0.0",
+    "generate-changelog": "^1.0.0",
+    "chai": "^3.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.2.5"
+  },
+  "dependencies": {
+    "hapi": "^13.5.0"
+  }
```
Comment (elnaz) [inline package.json]: nit: also the generator's fault (https://github.com/lob/generator/issues/11) but could you move `dependencies` above `devDependencies`?

**Comment by elnaz** [comment]
two nits but LGTM!

**Comment by dmlittle** [comment]
@elnaz done!

File: .node-version [hunk 3]
```diff
@@ -0,0 +1 @@
+5
```
Comment (mgartner) [inline .node-version]: Should we be using node 6 now?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+sudo: false
+node_js:
+  - 5
```
Comment (mgartner) [inline .travis.yml]: ditto re: node 6

File: config/staging.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+module.exports = {
+};
```
Comment (mgartner) [inline config/staging.js]: Staging should get a PORT, right?

File: config/production.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+module.exports = {
+};
```
Comment (mgartner) [inline config/production.js]: PORT?

File: package.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  "dependencies": {
+    "hapi": "^13.5.0"
+  },
+  "devDependencies": {
+    "eslint": "^1.9.0",
+    "eslint-config-lob": "^2.0.0",
+    "generate-changelog": "^1.0.0",
+    "chai": "^3.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.2.5"
```
Comment (mgartner) [inline package.json]: It's nice to keep this alphabetical, but not a huge deal, so up to you.

File: config/staging.js (hunk 5 above)
Comment (dmlittle) [inline config/staging.js]: In `03-server.md` it says we don't need to define `PORT` in `staging.js` and `production.js` right now. I left them blank assuming it wanted us to do it later on. Should we change that?

**Comment by robinjoseph08** [comment]
LGTM 👍

File: config/staging.js (hunk 5 above)
Comment (mgartner) [inline config/staging.js]: Ok, my bad.

**Pull request merged**